app/site/resources/view/templates/blocks/javascript.html.twig

```diff
@@ -1,11 +1,10 @@
 {% javascripts
 	'@Mothership:Site::resources/assets/js/jquery.1.11.1.min.js'
 	'@Mothership:Site::resources/assets/js/jquery.touch-swipe.js'
-	'@Mothership:Site::resources/assets/js/jquery.gallery.js'
 	'@Mothership:Site::resources/assets/js/jquery.menu.js'
-	'@Mothership:Site::resources/assets/js/jquery.carousel.js'
 	'@Mothership:Site::resources/assets/js/jquery.offcanvas.js'
 	'@Mothership:Site::resources/assets/js/jquery.basket.js'
+	'@Mothership:Site::resources/assets/js/ms-carousel.min.js'
 	'@Mothership:Site::resources/assets/js/jquery.setup.js'
 
 	output='/assets/js/ms-site-*.js'
```

Comment:
Can you reinstate the ?jsmin filter below. Probably my fault, but can you fix it as you are here?